group-owner.html

```diff
@@ -0,0 +1,314 @@
+  <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="move"></a>Moving data</h2>
+
+<p>When a user is removed from a group, any data they have in the group remains in the group, and is still owned by the user.</p>
+
+<p>Data can be moved to another group by the user before they are removed, or by the group owner or administrator at any time.</p>
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+<li><p class="first">Select <span class="bold">All Members</span> from the group display menu to view data owned by a user who is no longer a member of the group.</p>
+
+<p>Right-click on the data to be moved and select <span class="bold">Move to Group ... </span>.</p>
+
+<p>Select the destination from the pop-up window.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerMoveWeb.jpg" class="align-center" src="images/facilityManagerMoveWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<div class="line-block">
+  <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">Using the Command Line Interface (CLI), group owners (for data in their groups) and administrators (who are members of the groups) can:</p>
+
+<ul class="bullet-list">
+<li>view and move data owned by users no longer a member of a group - <a href="https://www.openmicroscopy.org/site/support/omero/users/cli/chgrp.html" target="_blank" class="external-url">chgrp</a></li>
+<li>change ownership of data.</li>
+</ul>
+
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</div><!-- End note -->
+
+<div class="line-block">
+  <div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="import"></a>Importing data for another user</h2>
+
+<p>Any data imported by a user will be owned by them and will be in a group of which they are a member. For details on importing see <a href="http://help.openmicroscopy.org/importing-data-5.html" target="_blank" class="external-url">Importing Data</a>.
+
+<p>Group Owners and Administrators, who are members of the group, can import data for any user in the group i.e. the data will belong to that user.</p>
```

Comment:
Sorry to be a pain. This sentence is now accurate for Insight. On the CLI, you can, being an admin, login as the (other) user himself, meaning you, being an admin do not have to be the member of the target group (only the user you are getting into like this has to be the member of the group of course). Really appreciate this is hard to explain in a succint form and not be confusing... There is no streamlining between CLI and Insight on this point.